### ISSUE TITLE
Missing elements

### ISSUE THREAD

**Issue by amanbayev** [rhyneav/papercss#62]
To celebrate 1k stars and to get ready for PaperCSS v2 (perhaps? 🥇 )
it would be nice to add following elements:
- [ ] datepicker
- [ ] sidebar, multilevel sidebar
- [ ] time picker
- [ ] icon font - would be awesome to have some icons (perhaps as a stand-alone package)
- [ ] using icons in form inputs - "input addons"
- [ ] calendar would be awesome
- [ ] timelines, wizards, steps
- [ ] sliders

p.s.: I really like how much work is done in paper css. But I believe adding this features would allow everyone to transfer one of their exisiting projects completely to PaperCSS.
In my case, I have an admin dashboard for sending newsletters that I am trying to style using papercss. Using other libraries, for datepickers for example, are odd-looking in the whole paper-theme.

**Comment by afzalsayed96** [comment]
Nice suggestions. I can work on the icons. Could you elaborate more how to go about it with the standalone package for icon font?

**Comment by valerymelou** [comment]
@amanbayev doesn't things like date picker, time picker, sidebar... will require JavaScript to work?

**Comment by amanbayev** [comment]
@valerymelou Yes, but I believe it is a common demand to have these JS components. I cannot imagine a web app without a datepicker in a form. The way bootstrap does it - they provide css and js file within a framework. What we could do is provide at least the CSS version of those components, and then other libraries could be used by serving custom styles. Almost all open-source components allow usage of custom styles. This way we can still work only on css/less part. But JS is not all that bad. Perhaps providing JS part would be great as well, to get started right away without relying on other 3rd party libraries much.

**Comment by amanbayev** [comment]
@afzalsayed96 perhaps using services like http://fontastic.me/ ?
from my understanding one needs to create a spritesheet with all the icons, and in css/less file it needs to be defined which class stands for which exact sprite in a spritesheet (starting position x,y, width and height etc.)
that those css/less and spritesheet files are included in the project to be used like <i class="[iconsetname] [iconset-iconname]"></i> or whatever could that classname be. at least that is how 100% of icon sets that I have used are organized and used.

**Comment by rhyneav** [comment]
Hmmm, maybe it's time to start thinking about components that involve JS. I'd like to keep it somewhat from the core of PaperCSS though. I think a lot of the value of this framework is being able to drop in the CSS file and be minimal with classes to achieve the paper look.

**Comment by antoninadert** [comment]
Soon we will have web components so it could involve js easily. Meanwhile we could go as standalone React components maybe. 

Having a css-only library is a must have though, I believe this is a very good approach and we should not pollute this library with js => Create another repo for js
